code/ZAS/Plasma.dm

```diff
@@ -119,19 +119,23 @@ obj/var/contaminated = 0
 		if(vsc.plc.GENETIC_CORRUPTION)
 			if(rand(1,10000) < vsc.plc.GENETIC_CORRUPTION)
 				randmutb(src)
-				src << "\red High levels of toxins cause you to spontaneously mutate."
+				src << "<span class='danger'>High levels of toxins cause you to spontaneously mutate.</span>"
 				domutcheck(src,null)
 
 
 /mob/living/carbon/human/proc/burn_eyes()
 	//The proc that handles eye burning.
-	if(prob(20)) src << "\red Your eyes burn!"
+	if(!species.has_organ["eyes"])
+		return
```

Comment:
Um, this is entirely pointless. ZAS is completely unused, files are only still around because no one has gotten to deleting them.